Dockerfile

```diff
@@ -4,4 +4,4 @@ COPY tests/tests.test .
 RUN mv tests.test /bin
 RUN apt-get update -y && apt-get install -y curl openssh-client git
 RUN curl -sSL http://deis.io/deis-cli/install-v2-alpha.sh | bash && mv ./deis /bin/deis
-CMD /bin/tests.test
+CMD ["/bin/tests.test", "-v", "-timeout=60m", "-ginkgo.v", "-ginkgo.slowSpecThreshold=120"]
```

Comment:
Would it be possible to move these into the Makefile as args to the `docker run` command issued by the `docker-test-integration` target? Something like:
```
	docker run -e DEIS_ROUTER_SERVICE_HOST=${DEIS_ROUTER_SERVICE_HOST} -e DEIS_ROUTER_SERVICE_PORT=${DEIS_ROUTER_SERVICE_PORT} ${IMAGE} ${TEST_OPTS}
```
If that would work, then we could have one canonical version (in the Makefile) of these options that both targets could consume.